lottie-lite.php

```diff
@@ -94,6 +95,12 @@
 		return $block_content;
 	}
 
+	// Allow themes/plugins to exclude specific blocks from lottie-lite's automatic handling
+	$excluded_blocks = apply_filters( 'lottie_lite_excluded_blocks', [] );
+	if ( in_array( $block['blockName'], $excluded_blocks, true ) ) {
+		return $block_content;
+	}
```

Comment:
Why would you have both a filter to set the supported blocks, and another to exclude them? The same filter that sets the supported blocks can be used here to achieve the same goal rather than creating a potentially very confusing situation.